### ISSUE TITLE
Make kaocha.stacktrace/*stacktrace-filters* configurable?

### ISSUE THREAD

**Issue by tonsky** [lambdaisland/kaocha#59]
In case I want to fine-tune them to my project?

**Comment by tonsky** [comment]
This was what I tried first too. Unfortunately exception is converted to string after test run is complete, so all bindings are already reset to their default value. For now I use this:

```
(alter-var-root
  #'kaocha.stacktrace/*stacktrace-filters*
  (constantly ["java." "clojure." "kaocha." "orchestra."]))
```

Would be great to have a config option to do this, as it seems a bit hacky

**Comment by plexus** [comment]
Fixed in #68
